web/yo/app/scripts/controllers/datasummary.js

```diff
@@ -387,110 +387,111 @@ angular.module('oncokbApp')
+                    }, {
                         attr: 'therapyLevels.3.num',
                         display: '# Therapy L-3'
-                    },{
+                    }, {
                         attr: 'therapyLevels.4.num',
                         display: '# Therapy L-4'
-                    },{
+                    }, {
                         attr: 'therapyLevels.r1.num',
                         display: '# Therapy L-R1'
-                    },{
+                    }, {
                         attr: 'therapyLevels.r2.num',
                         display: '# Therapy L-R2'
-                    },{
+                    }, {
                         attr: 'therapyLevels.r3.num',
                         display: '# Therapy L-R3'
-                    }]},
+                    }]
+                },
                 3: {
                     name: 'Treatment levels',
                     header: [{
                         attr: 'treatmentLevels.1.num',
                         display: '# Treatment L-1'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.2a.num',
                         display: '# Treatment L-2A'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.2b.num',
                         display: '# Treatment L-2B'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.3.num',
                         display: '# Treatment L-3'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.4.num',
                         display: '# Treatment L-4'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.r1.num',
                         display: '# Treatment L-R1'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.r2.num',
                         display: '# Treatment L-R2'
-                    },{
+                    }, {
                         attr: 'treatmentLevels.r3.num',
                         display: '# Treatment L-R3'
-                    }]}
+                    }]
+                }
             };
             var headers = [];
 
-            if(tableKeys.hasOwnProperty(tableId.toString())){
+            if (tableKeys.hasOwnProperty(tableId.toString())) {
                 var _headers = [];
-                switch(type){
-                    case 'csv':
-                        downloadOpts.filename += 'csv';
-                        seperator = ',';
-                        break;
-                    case 'tsv':
-                        downloadOpts.filename += 'tsv';
-                        seperator = '\t';
-                        break;
-                    default:
-                        downloadOpts.filename += 'tsv';
-                        seperator = '\t';
-                        break;
+                switch (type) {
```

Comment:
Nope, we no longer use indent for case. You can take a look at this: http://javascript.crockford.com/code.html